containers/src/Kokkos_DynRankView.hpp

```diff
@@ -478,9 +483,17 @@ class DynRankView : private View<DataType*******, Properties...> {
   // Types below - at least the HostMirror requires the value_type, NOT the rank
   // 7 data_type of the traits
 
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
   /** \brief  Compatible view of array of scalar types */
-  using array_type = DynRankView<
-      typename drvtraits::scalar_array_type, typename drvtraits::array_layout,
+  using array_type KOKKOS_DEPRECATED_WITH_COMMENT(
+      "Use Kokkos_DynRankView::type insted.") =
```

Comment:
```suggestion
      "Use type instead.") =
```

Reply:
Or `Kokkos::DynRankView::type` but the prefix is superfluous.  Not the inste[a]d typos.